### PR TITLE
Changed the label information

### PR DESCRIPTION
**Pull request by clemenko**
Updated the label information to include the objects that it can be applied to.

### CODE REVIEW

File: datacenter/ucp/2.0/guides/user-management/permission-levels.md [hunk 1]
```diff
@@ -43,8 +43,7 @@ users can see the containers they deploy in the cluster.
 
 ## Team permission levels
 
-Teams allow you to define fine-grain permissions to services, containers, and
-networks that have the label `com.docker.ucp.access.label` applied to them.
+Teams and Labels give the administrator fine-grained control over permissions. Each team can have multiple labels. Each label has a key of `com.docker.ucp.access.label`. The label is then applied to the containers, services, networks, secrets and volumes.
```
Comment (joaofnfernandes) [inline datacenter/ucp/2.0/guides/user-management/permission-levels.md]: Can we also make it explicit what resources don't have RBAC? Is it just images and nodes @alexmavr?

Comment (joaofnfernandes) [inline datacenter/ucp/2.0/guides/user-management/permission-levels.md]: Make "Labels" lower case. Only product names should be capitalized :)

Comment (alexmavr) [inline datacenter/ucp/2.0/guides/user-management/permission-levels.md]: RBAC does not currently apply for images, nodes, plugins and ports

**Comment by clemenko** [comment]
Updated branch.

**Pull request merged**